sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -1573,7 +1575,7 @@ udf_aggregates AS (
     aggregates
 )
 SELECT
-  * EXCEPT (map_sum_aggregates),
+  * EXCEPT (profile_group_id, map_sum_aggregates),
```

Comment:
I think you don't need to explicitly exclude and add `profile_id_group` at the end here. BigQuery will add this column automatically to the end of the existing table schema

Reply:
Good to know, thanks! Updated the query, please re-review when you get time